apps/modernization-ui/src/pages/patient/search/result/PatientResult.tsx

```diff
@@ -119,7 +119,7 @@ const ResultItem = ({ label, orientation = 'horizontal', children }: ResultItemP
 type ResultItemLabelProps = { children: string };
 
 const ResultItemLabel = ({ children }: ResultItemLabelProps) => (
-    <span className="patient-search-result-item-label">{children}</span>
+    <span className="patient-search-result-item-label font-sans-3xs">{children}</span>
```

Comment:
question:  Why was the font size here changed?  this value seems small.

Reply:
I changed that font size to ensure for the children that we get the same font size across the summary card for the headers making it smaller but visually thought it looked good, will play around with a bigger size to see if doable.